src/index.html

```diff
@@ -7,34 +7,34 @@
   </head>
   <body>
     <header>
-      <h1>Leaderboard</h1>
+      <h1><span>PAC-MAN</span> Leaderboard</h1>
     </header>
-    <section class="refresh">
-      <h2>Recent scores</h2>
-      <button id="refresh" type="button">Refresh</button>
-    </section>
-    <table class="leaderboard">
-      <tr>
-        <td>No players found</td>
-      </tr>
-    </table>
-    <form>
-      <h2>Add your score</h2>
-      <input
-        type="text"
-        id="fname"
-        name="fname"
-        placeholder="Your name"
-        required
-      />
-      <input
-        type="number"
-        id="score"
-        name="score"
-        placeholder="Your score"
-        required
-      />
-      <button type="submit">Submit</button>
-    </form>
+    <main>
+      <section>
+        <div class="leaderboard-heading">
+          <h2>Recent scores</h2>
+          <button id="refresh" class="btn" type="button">Refresh</button>
+        </div>
+        <ul class="leaderboard"></ul>
+      </section>
+      <form>
+        <h2>Add your score</h2>
+        <input
+          type="text"
+          id="fname"
+          name="fname"
+          placeholder="Your name"
+          required
+        />
+        <input
+          type="number"
+          id="score"
+          name="score"
+          placeholder="Your score"
+          required
+        />
+        <button class="btn" type="submit">Submit</button>
+      </form>
```

Comment:
Make sure that your design keeps the general layout of the wireframe to meet the project requirements 🙌 Kindly make the necessary changes in your design so that your unordered list and the form level up vertically 📝 You have done an outstanding job implementing the styling in your leaderboard 🔥😎

### Your design 
![image](https://user-images.githubusercontent.com/15807343/151011708-4e60519a-930d-43cd-b784-6f72aa9914c9.png)

### Wireframe
![image](https://user-images.githubusercontent.com/15807343/151012319-8cd60254-29a5-40e0-84c3-15bd55d16d7d.png)